### ISSUE TITLE
PHP Fatal error:  Uncaught Error: Class 'Ajax\php\ubiquity\JsUtils' not found in .....vendor/phpmv/ubiquity/src/Ubiquity/core/Framework.php:87

### ISSUE THREAD

**Issue by EDUILSON** [phpMv/ubiquity#22]
I faithfully followed the quick start of the documentation. Installation occurred smoothly. The server runs smoothly using the **"Ubiquity serve"** command. But when trying to access the site the console prints this error: 

> PHP Fatal error:  Uncaught Error: Class 'Ajax\php\ubiquity\JsUtils' not found in /home/eduilson/Documents/als/ubiquity/vendor/phpmv/ubiquity/src/Ubiquity/core/Framework.php:87

**Ubiquity version** output:
    **PHP** 7.2.4-1+b1
    **Ubiquity devtools** (1.2.0) 
    **Ubiquity** 2.1.0

**Comment by jcheron** [comment]
In `app/config/config.php`,
di key must be set to:
```php
"di"=>["@exec"=>["jquery"=>function($controller){
				return \Ubiquity\core\Framework::diSemantic($controller);
	}
]],
```
It looks like this value has been changed, I do not know why...

**Comment by jcheron** [comment]
Fixed in [devtools 1.2.2](https://github.com/phpMv/ubiquity-devtools/releases/tag/v1.2.2)
Run:
```
composer global update
```
for installing the last devtools version
